Site/ASAB Maestro/Descriptors/asab-iris.yaml

```diff
@@ -19,6 +19,8 @@ asab:
     # [smtp] must be filled from general confguration (user)
     # [slack] must be filled from general confguration (user)
     # [variables] must be filled from general confguration (user)
+  email:
+    markdown_wrapper: /Templates/Wrapper/markdown_wrapper.html
```

Comment:
Wrong name of the template.

Reply:
fixed here and in lmio-common-library. I still don't like the reference here to lmio common library. At the same time, I don't want to create any new logic...